hiera.yaml

```diff
@@ -12,3 +12,5 @@ hierarchy:
     path: '%{facts.os.name}-%{facts.os.release.major}.yaml'
   - name: 'Distribution Name'
     path: '%{facts.os.name}.yaml'
+  - name: 'OS Family Major Version'
+    path: '%{facts.os.family}-%{facts.os.release.major}.yaml'
```

Comment:
I suspect, having os.family and os.name on the same level isn't very clever.

Reply:
For EL systems you almost always want to use `os.family` because RedHat, CentOS, Scientific, Oracle, and now Alma and Rocky Linux will all respond as `os.family = RedHat` and they can almost always be treated the same since they are all rebuilds of RedHat.  The `os.name` here is going to get used before `os.family-os.release.major` because it's listed first in this hierarchy, and the `os.name-os.release.major` for is going to be what gets used for Debian, while this one I added will be used for EL systes.

Reply:
I think, os.family should also fit for Debian.